src/types/index.ts

```diff
@@ -1,18 +1,29 @@
+export enum IntegrationId {
```

Comment:
I also debated including these or not, but decided that since we update integrations so rarely, it should be fine. I also like this since it'll allow us to expose integrations as a beta feature by essentially telling specific customers what the Id is without exposing it here. In that way we can declaratively specify in code "hey these are the integrations we claim to fully support"

Reply:
Actually I think I'm going to remove these. The reason being that we should encourage people to use the endpoint for getting integrations rather than anything static. This will allow us to turn off the integration if it breaks for whatever reason.